src/Fixer/ControlStructure/YodaStyleFixer.php

```diff
@@ -90,6 +90,19 @@ public function getDefinition()
                         'always_move_variable' => true,
                     ]
                 ),
+                new CodeSample(
+                    '<?php
+    // Enforce regular, non-Yoda conditions.
```

Comment:
Let's drop the word "regular" as it suggests that Yoda conditions are irregular which sounds somehow negative.

Reply:
Agreed. Removing it now...

Reply:
Done https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5293/commits/f83ea0567a9ac685bac7b856e14e59a5e5287f9c